### PR TITLE
feat/TinderStyleVoting 

### PR DESCRIPTION
**Pull request by VAIBHAV-25**

### CODE REVIEW

File: src/components/AppLayout/index.tsx [hunk 1]
```diff
@@ -1110,7 +1110,7 @@ const AppLayout = ({ className, Component, pageProps }: Props) => {
 	}
 
 	return (
-		<Layout className={className}>
+		<Layout className={`${className} overflow-x-hidden overflow-y-hidden`}>
```
Comment (KanishkaRajputd) [inline src/components/AppLayout/index.tsx:1113]: overflow none?

File: src/components/AppLayout/index.tsx [hunk 2]
```diff
@@ -1200,7 +1200,7 @@ const AppLayout = ({ className, Component, pageProps }: Props) => {
 						</div>
 					</Layout>
 				) : (
-					<Layout className={'min-h-[calc(100vh - 10rem)] flex flex-row bg-[#F5F6F8] dark:bg-section-dark-background'}>
+					<Layout className={'min-h-[calc(100vh - 10rem)] overflow-x-none overflow-y-none flex flex-row bg-[#F5F6F8] dark:bg-section-dark-background'}>
```
Comment (KanishkaRajputd) [inline src/components/AppLayout/index.tsx:1203]: overflow none?

File: src/components/Listing/Tracks/AboutTrackCard.tsx [hunk 3]
```diff
@@ -552,4 +551,17 @@ export default styled(AboutTrackCard)`
 			display: block !important;
 		}
 	}
+	.image-button-wrapper {
```
Comment (KanishkaRajputd) [inline src/components/Listing/Tracks/AboutTrackCard.tsx:554]: are we using these classes?

File: src/components/Post/GovernanceSideBar/Referenda/VoteReferendumCard.tsx [hunk 4]
```diff
@@ -0,0 +1,473 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
```
Comment (KanishkaRajputd) [inline src/components/Post/GovernanceSideBar/Referenda/VoteReferendumCard.tsx:1]: this should be under tinder style folder

File: src/components/Post/GovernanceSideBar/Referenda/VotingFormCard.tsx [hunk 5]
```diff
@@ -0,0 +1,118 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
```
Comment (KanishkaRajputd) [inline src/components/Post/GovernanceSideBar/Referenda/VotingFormCard.tsx:1]: this should be under tinder style folder

File: src/components/VoteCart/ProposalInfoCard.tsx [hunk 6]
```diff
@@ -0,0 +1,285 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
```
Comment (KanishkaRajputd) [inline src/components/VoteCart/ProposalInfoCard.tsx:3]: this should be under tinder style folder

File: src/components/VoteCart/VoteSuccessModal.tsx [hunk 7]
```diff
@@ -0,0 +1,41 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
```
Comment (KanishkaRajputd) [inline src/components/VoteCart/VoteSuccessModal.tsx:1]: this should be under tinder style folder

File: src/components/VoteCart/index.tsx [hunk 8]
```diff
@@ -0,0 +1,234 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+import { Button, Modal, Spin } from 'antd';
+import React, { useEffect, useState } from 'react';
+import { useBatchVotesSelector, useNetworkSelector, useUserDetailsSelector } from '~src/redux/selectors';
```
Comment (KanishkaRajputd) [inline src/components/VoteCart/index.tsx:6]: this should be under tinder style folder

File: src/components/VotingCards/CardComponents/CardComments.tsx [hunk 9]
```diff
@@ -0,0 +1,39 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+import React, { FC } from 'react';
+import ImageIcon from '~src/ui-components/ImageIcon';
+import { ICardComments } from '../types';
```
Comment (KanishkaRajputd) [inline src/components/VotingCards/CardComponents/CardComments.tsx:6]: this should be under tinder style folder

File: src/components/VotingCards/CardComponents/CartOptionMenu.tsx [hunk 10]
```diff
@@ -0,0 +1,67 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
```
Comment (KanishkaRajputd) [inline src/components/VotingCards/CardComponents/CartOptionMenu.tsx:2]: this should be under tinder style folder

File: src/components/VotingCards/CardComponents/SwipeActionButtons.tsx [hunk 11]
```diff
@@ -0,0 +1,47 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+import React, { FC } from 'react';
+import LikeWhite from '~assets/icons/like-white.svg';
```
Comment (KanishkaRajputd) [inline src/components/VotingCards/CardComponents/SwipeActionButtons.tsx:5]: this should be under tinder style folder

File: src/components/VotingCards/CardComponents/TinderCardsComponent.tsx [hunk 12]
```diff
@@ -0,0 +1,75 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+import React, { FC, useState } from 'react';
+import CardPostHeading from '../PostInfoComponents/CardPostHeading';
+import { Button, Divider } from 'antd';
+import Markdown from '~src/ui-components/Markdown';
+// import ReferendumV2VoteInfo from '../Post/GovernanceSideBar/Referenda/ReferendumV2VoteInfo';
+import { IVotesCount } from '~src/types';
```
Comment (KanishkaRajputd) [inline src/components/VotingCards/CardComponents/TinderCardsComponent.tsx:9]: this should be under tinder style folder

File: src/components/VotingCards/PostInfoComponents/CardPostDescription.tsx [hunk 13]
```diff
@@ -0,0 +1,30 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
```
Comment (KanishkaRajputd) [inline src/components/VotingCards/PostInfoComponents/CardPostDescription.tsx:2]: this should be under tinder style folder

File: src/components/VotingCards/PostInfoComponents/CardPostHeading.tsx [hunk 14]
```diff
@@ -0,0 +1,378 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { Divider } from 'antd';
+import { dayjs } from 'dayjs-init';
+import dynamic from 'next/dynamic';
+import { useRouter } from 'next/router';
+import React, { FC, useEffect, useState } from 'react';
+import { noTitle } from 'src/global/noTitle';
```
Comment (KanishkaRajputd) [inline src/components/VotingCards/PostInfoComponents/CardPostHeading.tsx:10]: this should be under tinder style folder

File: src/components/VotingCards/PostInfoComponents/CardPostInfo.tsx [hunk 15]
```diff
@@ -0,0 +1,113 @@
...
+import React, { FC } from 'react';
+import PostOnChainInfo from '../../Post/Tabs/PostOnChainInfo';
+import { isOffChainProposalTypeValid } from '~src/api-utils';
+import CardPostHeading from './CardPostHeading';
+import { useTheme } from 'next-themes';
+import CardPostDescription from './CardPostDescription';
+import { Tabs } from '~src/ui-components/Tabs';
+
+interface ICardPostInfo {
+	post: any;
```
Comment (KanishkaRajputd) [inline src/components/VotingCards/PostInfoComponents/CardPostInfo.tsx:13]: this should be under tinder style folder

File: src/components/VotingCards/PostInfoComponents/ReferendumV2CardInfo.tsx [hunk 16]
```diff
@@ -0,0 +1,407 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { Modal, Spin } from 'antd';
+import { LoadingOutlined, InfoCircleOutlined, LikeFilled } from '@ant-design/icons';
```
Comment (KanishkaRajputd) [inline src/components/VotingCards/PostInfoComponents/ReferendumV2CardInfo.tsx:6]: this should be under tinder style folder

File: src/components/VotingCards/index.tsx [hunk 17]
```diff
@@ -0,0 +1,233 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+import React, { useEffect, useMemo, useRef, useState } from 'react';
+import TinderCard from 'react-tinder-card';
+import { LeftOutlined, RightOutlined } from '@ant-design/icons';
+import { batchVotesActions } from '~src/redux/batchVoting';
+import { useAppDispatch } from '~src/redux/store';
+import { useBatchVotesSelector, useNetworkSelector, useUserDetailsSelector } from '~src/redux/selectors';
+import SwipeActionButtons from './CardComponents/SwipeActionButtons';
```
Comment (KanishkaRajputd) [inline src/components/VotingCards/index.tsx:10]: this should be under tinder style folder

File: src/components/VotingCards/types/index.ts [hunk 18]
```diff
@@ -0,0 +1,51 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
```
Comment (KanishkaRajputd) [inline src/components/VotingCards/types/index.ts:4]: this should be under tinder style folder

File: src/components/AppLayout/index.tsx (hunk 1 above)
Comment (VAIBHAV-25) [reply src/components/AppLayout/index.tsx:1113]: woh cards swipe k time x and y direction se space leraha tha so overflow band kr diya

File: src/components/AppLayout/index.tsx (hunk 2 above)
Comment (VAIBHAV-25) [reply src/components/AppLayout/index.tsx:1203]: woh cards swipe k time x and y direction se space leraha tha so overflow band kr diya

**Pull request merged**